app/views/templates/ChangeSuccessView.scala.html

```diff
@@ -0,0 +1,64 @@
+@*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import models.contactPreferences.ContactPreference
+@import models.viewModels.ChangeSuccessViewModel
+
+@this(mainTemplate: MainTemplate, updateClient: UpdateClient)
+
+@(model: ChangeSuccessViewModel)(implicit request: Request[_], messages: Messages, appConfig: config.AppConfig, user: User[_])
+
+@mainTemplate(title = messages(model.pageTitle)) {
+
+<div class="govuk-box-highlight">
+  <h1 class="heading-xlarge">@messages(model.pageTitle)</h1>
+</div>
+
+<h2 class="heading-medium">@messages("common.whatHappensNext")</h2>
+
+@if(user.isAgent) {
+
+  @model.agentEmail match {
+    case Some(email) => {<p>@messages("changeSuccess.helpOne.digital.agent", email)</p>}
+    case _ => {<p>@messages("changeSuccess.helpOne.paper.agent")</p>}
+  }
+
+  @updateClient(model.businessName)
+
+} else {
+
+  @model.preference match {
+
+    case (Some(ContactPreference.digital)) => {
+      <p id="preference-message">@messages("changeSuccess.helpOne.digital")</p>
+    }
+    case (Some(ContactPreference.paper)) => {
+      <p id="preference-message">@messages("changeSuccess.helpOne.paper")</p>
+    }
+    case _ => {
+      <p>@messages("common.helpOne.apiFailure")</p>
+    }
+  }
+
+  <p>@messages("changeSuccess.helpTwo")</p>
+}
+
+@if(user.isAgent) {
+  <p><a href="@appConfig.vatAgentClientLookupServicePath">@messages("common.changeClient")</a></p>
+}
```

Comment:
This p can go in the if statement above to avoid having to have another if statement.